src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards | Home page</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body class="content">
+    <div class="product-card" data-qa="card">
+
+      <img
+       class="product-card__image"
+       src="./images/imac.jpeg"
+       alt="iMac"
+      >
+
+      <div class="description product-card__description">
+        <div class="description description__product--name">
```

Comment:
i don't understand why you use the modifier here  <div class="description description__product--name">  why you use modifier instead block elements
because i was thinking that we need a modifier to change the view of block or element but not change the whole structure and style
but to be honest i know BEM bad so be happy if you will tell me

Reply:
> i don't understand why you use the modifier here
> 
> why you use modifier instead block elements
> because i was thinking that we need a modifier to change the view of block or element but not change the whole structure and style
> but to be honest i know BEM bad so be happy if you will tell me

That's only one way that Bem give me to start npm, maybe you right. I will check one more time